ui/pages/bridge/prepare/prepare-bridge-page.tsx

```diff
@@ -214,8 +217,27 @@ const PrepareBridgePage = () => {
   }, [rotateSwitchTokens]);
 
   useEffect(() => {
-    // Reset controller and inputs on load
-    dispatch(resetBridgeState());
+    if (activeQuote) {
```

Comment:
Do we only want to restore if there's an active quote? It's possible to close the Extension before we get a quote and they still lose their progress.

Reply:
The ticket here was specific to clicking on the ToS link, which only shows in the quote card that appears after the quotes finish loading